RobotCode/src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -88,11 +84,13 @@ public void periodic()
                 intakeRollerPidController.setReference(desiredVelocity, ControlType.kVelocity, 0);
         }
 
+        /** Ticks */
```

Comment:
Do `/** @param desiredRotation Ticks */`

Reply:
Also specify what range this should be in

Reply:
The range is currently unknown and will be known by constants as there will be a constant for the minimum (retracted) position and maximum (deployed) position.

Reply:
Ok, that's fine